docs/cli/getting-started.md

```diff
@@ -23,6 +23,7 @@ With [v0.14.0](https://github.com/vmware-tanzu/tanzu-framework/releases/tag/v0.1
 The API driven plugin discovery feature is enabled as default method to install the plugins. So, most users will just need to install the Tanzu CLI binary. The plugins can be installed and configured with `tanzu plugin sync` command. Learn more about this feature in the [design docs](../design/context-aware-plugin-discovery-design.md).
 
 ##### macOS/Linux
```

Comment:
The markdown linter isn't going to like this two headers together like this. I don't think content is required, but it will at least need a blank line between them.

Reply:
Have added blank lines between the two headers. Running the linter now seems to be green. Please let me know if I have misinterpreted your comment here.